prog/vm/host_nexus.rb

```diff
@@ -5,18 +5,20 @@ class Prog::Vm::HostNexus < Prog::Base
 
   def self.assemble(sshable_hostname, location: "hetzner-hel1", net6: nil, ndp_needed: false, provider: nil, hetzner_server_identifier: nil)
     DB.transaction do
-      sa = Sshable.create_with_id(host: sshable_hostname)
+      ubid = UBID.generate(UBID::TYPE_VM_HOST)
```

Comment:
Seems like we ought to have a `VmHost.generate_ubid` (or something) but that's for another time (if at all)

Reply:
Yeah. I used it first. But vm host table has foreign key for its id column. Because of that we need to sshable first

Reply:
```

  create_table(:vm_host) do
      foreign_key :id, :sshable, type: :uuid, primary_key: true
      column :allocation_state, :allocation_state, default: "unprepared", null: false
      column :ip6, :inet, unique: true
      column :net6, :cidr, unique: true
    end

```

Reply:
https://github.com/ubicloud/ubicloud/pull/373